db/migrate/20190403233704_create_user_languages.rb

```diff
@@ -0,0 +1,10 @@
+class CreateUserLanguages < ActiveRecord::Migration[5.2]
+  def change
+    create_table :user_languages do |t|
+      t.integer :user_id
+      t.integer :language_id
```

Comment:
Both of these columns should be references with foreign key constraints, similar to the example at the end of this section of the Rails Guides: https://guides.rubyonrails.org/active_record_migrations.html#creating-a-standalone-migration. 

Alternatively, you can create a migration specifically designed for making a new join table: https://guides.rubyonrails.org/active_record_migrations.html#creating-a-join-table. 

Since you've already run this migration on your local machine, you will need to 
1. rollback your most recent migration with `rails db:rollback`
2. delete this migration file
3. create a new migration file (I recommend using `create_join_table :users, :languages` and then adding in the `level` column to the generated migration file)
4. run the new migration with `rails db:migrate`